docker/cassandra/default-config/logback.xml

```diff
@@ -1,78 +1,16 @@
-
-    <appender name="SYSTEMLOG" class="ch.qos.logback.core.rolling.RollingFileAppender">
-        <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
-            <level>INFO</level>
-        </filter>
-        <file>${cassandra.logdir}/system.log</file>
-        <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
-            <fileNamePattern>${cassandra.logdir}/system.log.%i.zip</fileNamePattern>
-            <minIndex>1</minIndex>
-            <maxIndex>20</maxIndex>
-        </rollingPolicy>
-        <triggeringPolicy class="ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy">
-            <maxFileSize>20MB</maxFileSize>
-        </triggeringPolicy>
-        <encoder>
-            <pattern>%-5level [%thread] %date{ISO8601} %F:%L - %msg%n</pattern>
-        </encoder>
-    </appender>
-
-    <!-- DEBUGLOG rolling file appender to debug.log (all levels) -->
-
-    <appender name="DEBUGLOG" class="ch.qos.logback.core.rolling.RollingFileAppender">
-        <file>${cassandra.logdir}/debug.log</file>
-        <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
-            <fileNamePattern>${cassandra.logdir}/debug.log.%i.zip</fileNamePattern>
-            <minIndex>1</minIndex>
-            <maxIndex>20</maxIndex>
-        </rollingPolicy>
-        <triggeringPolicy class="ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy">
-            <maxFileSize>20MB</maxFileSize>
-        </triggeringPolicy>
-        <encoder>
-            <pattern>%-5level [%thread] %date{ISO8601} %F:%L - %msg%n</pattern>
-        </encoder>
-    </appender>
-
-    <!-- ASYNCLOG assynchronous appender to debug.log (all levels) -->
-
-    <appender name="ASYNCDEBUGLOG" class="ch.qos.logback.classic.AsyncAppender">
-        <queueSize>1024</queueSize>
-        <discardingThreshold>0</discardingThreshold>
-        <includeCallerData>true</includeCallerData>
-        <appender-ref ref="DEBUGLOG" />
-    </appender>
-
-    <!-- STDOUT console appender to stdout (INFO level) -->
-
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
-            <level>INFO</level>
-        </filter>
         <encoder>
-            <pattern>%-5level [%thread] %date{ISO8601} %F:%L - %msg%n</pattern>
+            <pattern>%-5level [%thread] %F:%L %msg%n</pattern>
         </encoder>
     </appender>
 
-    <!-- Uncomment bellow and corresponding appender-ref to activate logback metrics
-    <appender name="LogbackMetrics" class="com.codahale.metrics.logback.InstrumentedAppender" />
-     -->
-
     <root level="INFO">
-        <appender-ref ref="SYSTEMLOG" />
         <appender-ref ref="STDOUT" />
-        <appender-ref ref="ASYNCDEBUGLOG" /> <!-- Comment this line to disable debug.log -->
-        <!--
-        <appender-ref ref="LogbackMetrics" />
-        -->
     </root>
 
     <logger name="org.apache.cassandra" level="DEBUG"/>
```

Comment:
looks like we are now including debug logging by default as well (found in testing)